src/wp-includes/class-wp-theme-json.php

```diff
@@ -1925,6 +1925,53 @@ public function get_styles_block_nodes() {
 		return static::get_block_nodes( $this->theme_json );
 	}
 
+	/**
+	 * Returns a filtered declarations array if there is a separator block with only a background
+	 * style defined in theme.json by adding a color attribute to reflect the changes in the front.
+	 *
+	 * @since 6.1.1
+	 *
+	 * @param array $declarations List of declarations.
+	 *
+	 * @return array $declarations List of declarations filtered.
+	 */
+	private static function update_separator_declarations( $declarations ) {
+		$background_matches = array_values(
+			array_filter(
+				$declarations,
+				function( $declaration ) {
```

Comment:
When using a closure as a callback, add `static` for a tiny performance micro-optimization:

```suggestion
				static function( $declaration ) {
```